python/lsst/ts/hvac/hvac_csc.py

```diff
@@ -305,6 +323,12 @@ def _do_enable(self, topic, data):
             json.dumps(data.comandoEncendido),
         )
         self.log.info(f"{topic.name} enable command sent {is_published}")
+        if is_published:
```

Comment:
Please clarify the log message a bit. I think it will be "foo enable command sent True" which I find confusing (what does it mean to send True or False?). Please consider a warning if the command fails. For example:
```
if is_published:
    self.log.info(f"{topic_name} enabled")
else:
    self.log.warning(f"{topic_name} enable failed")
```

I gently suggest a clearer name than `is_published`, such as `succeeded`, `success`, `isok`, or even `was_published`. It is telling you whether the message was published.

Reply:
I renamed the return value to was_published, even if it sounds strange to me but that may be due to my lack of English. I also removed the log message since I added it for debugging purposes only.